apps/src/lib/ui/lti/sync/LtiSectionSyncDialog.tsx

```diff
@@ -56,6 +59,14 @@ export default function LtiSectionSyncDialog({
   };
 
   const disableRosterSyncView = () => {
+    const eventPayload = {
+      lms_type: lmsType,
+    };
+    analyticsReporter.sendEvent(
+      'lti_opt_out_dialogue',
```

Comment:
Do we have a preference for naming these events with a verb, like how the other opt out events are `toggle_on`, `toggle_off`, and `confirm`? If so this one should probably be `lti_opt_out_dialogue_view` or  `lti_opt_out_dialogue_show`

Reply:
I was working off Ali's [doc](https://docs.google.com/document/d/19Yxl73BAA7lsA_s4s27rD2ITuvyb4U39iVSZETlDLOs/edit#heading=h.gdpxre740sd2), however I want to circle back with him on some of these naming conventions. Even camel_casing these is a question, as I've seen other logs have just normal words with spacing.

Reply:
I chatted with Ali a little this morning. Since he's been out, I haven't had a chance to sync with him on naming these events. I don't want to block this PR any longer, so I'd like to merge with the names as is, and make any potential changes in a follow-up PR.